GimbalAdapter.swift

```diff
@@ -47,49 +50,51 @@ open class GimbalAdapter {
      * Restores the adapter. Should be called in didFinishLaunchingWithOptions.
      */
     open func restore() {
-        if (GMBLPlaceManager.isMonitoring()) {
-            start(nil);
-        }
+        isStarted = Gimbal.isStarted()
+        placeManager.delegate = gimbalDelegate
```

Comment:
in the other methods if gimbal is not started the delegate is not set. Should this be:
```
if (Gimbal.isStarted()) {
  isStarted = true
  placeManager.delegate = gimbalDelegate
  setDeviceAttributes()
}
```